sdk/src/androidTest/assets/hcaptcha-form.html

```diff
@@ -17,7 +17,7 @@
 
         function onHcaptchaLoaded() {
             try {
-                BridgeObject.onLoaded();
+                BridgeObject.onOpen();
```

Comment:
The `onLoaded` should still be kept. The new `onOpen` shall be added to the `getRenderConfig` line 135:

```js
'open-callback': function openCallback() {
    return BridgeObject.onLoaded();
}
```

Reply:
Oh, this is test mock file.